avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -442,6 +443,8 @@ private static Object fromConnectData(Schema schema, org.apache.avro.Schema avro
             return maybeAddContainer(avroSchema, converted, requireContainer);
           } else {
             List<GenericRecord> converted = new ArrayList<>(map.size());
+            org.apache.avro.Schema underlyingAvroSchema =
+                avroSchemaForUnderlyingMapEntryType(schema, avroSchema);
```

Comment:
Minor nit: might be a bit more readable to have this on one line like L434 above.